tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -169,3 +169,180 @@ test.describe('Choose Template Type Page', () => {
     await expect(chooseTemplatePage.goBackLink).toBeHidden();
   });
 });
+
+test.describe('NHS App Message Template tests -in progress', () => {
+  const sessionStorageHelper = new SessionStorageHelper([
+    emptySessionData,
+    emptySessionDataForRadioSelect,
+    nhsAppRadioSelectedSessionData,
+  ]);
+
+  test.beforeAll(async () => {
+    await sessionStorageHelper.seedSessionData();
```

Comment:
you are seeding and deleting the same session data twice in two different test suites - this is not safe because when the first test suite ends, it will delete the session data so that the second suite cannot use it.